src/plugins/prefs/autoconfig/easyTestSetup.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+TriggerPort=65432
+MountPoint="user/prefs"
+ConfigFile="myPrefs.js"
+Decoy="/tmp/imnotreal.js"
+AutoConfigScript="./firefox.cfg"
```

Comment:
Assumes the working directory to be `autoconfig`, which most likely is not the case.